torch/testing/_internal/common_methods_invocations.py

```diff
@@ -588,22 +588,38 @@ def sample_inputs_linalg_vector_norm(op_info, device, dtype, requires_grad, **kw
 
     return inputs
 
+def sample_inputs_addmm_non_fusible_nodes(op_info, device, dtype, requires_grad, **kwargs):
+    for_inplace_variant = kwargs.get('for_inplace_variant', False)
```

Comment:
I think the signature can be written as `..., *, for_inplace_variant=False, **kwargs)`